### ISSUE TITLE
cover-bg-crossfade - respect prefers-reduced-motion

### ISSUE THREAD

**Issue by markhowellsmead** [a8cteam51/special-projects-blocks-monorepo#149]
I'd suggest that you extend cover-bg-crossfade so that it respects the user's preference for reduced motion.

**Comment by nattarnoff** [comment]
Seconding this. As a person with a vestibular disorder, I can attest that the images used in the demo video are triggering to those sensitive to motion. Depending on what an author chooses this can cause anxiety, migraines, dizziness, nausea, physical exhaustion, and brain fog. 

Ideally with the setting of `reduced` the images scroll with their associated block. If not possible, Image fade to black fade to new image will be easier for users. Solid color fades will be less problematic.

**Comment by dhanson-wp** [comment]
This is great feedback! We'll get this on our roadmap. Thank you!
